contributors.md

```diff
@@ -1,4 +1,4 @@
-# Contributors to OpenRCT2
+﻿# Contributors to OpenRCT2
```

Comment:
I think you may have unintentionally inserted a byte order mark here. You must explicitly save as UTF-8 without BOM in your editor.